src/utils/bwellPersonFinder.js

```diff
@@ -0,0 +1,77 @@
+const {assertTypeEquals} = require('./assertType');
+const {DatabaseQueryFactory} = require('../dataLayer/databaseQueryFactory');
+
+class BwellPersonFinder {
+    /**
+     * constructor
+     * @param {DatabaseQueryFactory} databaseQueryFactory
+     */
+    constructor(
+        {
+            databaseQueryFactory
+        }
+    ) {
+        /**
+         * @type {DatabaseQueryFactory}
+         */
+        this.databaseQueryFactory = databaseQueryFactory;
+        assertTypeEquals(databaseQueryFactory, DatabaseQueryFactory);
```

Comment:
I'm not aware of a particular need for it, but I was following the pattern from other classes (e.g., AdminPersonPatientLinkManager). maybe that's not the pattern I should be following in this case?

Reply:
I seen it in other places too, but not sure is it a pattern that we've established either.

Reply:
I do like the idea of testing that all in the container, rather than in every class that consumes from the container. I think that's out of scope for this ticket though.